katamari/com.hack_computer.Clubhouse.json.in

```diff
@@ -21,6 +21,8 @@
         "--env=GSETTINGS_SCHEMA_DIR=/run/host/usr/share/glib-2.0/schemas/",
         "--env=GTK3_MODULES=/app/clippy/lib/libclippy-module.so",
         "--filesystem=/var/lib/AccountsService/icons/:ro",
+        "--filesystem=/usr/share/applications:ro",
+        "--filesystem=/var/lib/flatpak/exports/share/applications:ro",
```

Comment:
It seems to me that the exposed filesystem folders don't quite match the data_dirs being looked above. I would make sure that only exposed folders are looked for desktop files.